common/messaging/kafka/kafkaClient.go

```diff
@@ -18,43 +18,44 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package messaging
+package kafka
 
 import (
 	"crypto/tls"
 	"crypto/x509"
 	"fmt"
 	"io/ioutil"
 	"strings"
-
-	"github.com/uber/cadence/common/auth"
+	"time"
 
 	"github.com/Shopify/sarama"
-	uberKafkaClient "github.com/uber-go/kafka-client"
-	uberKafka "github.com/uber-go/kafka-client/kafka"
 	"github.com/uber-go/tally"
-	"go.uber.org/zap"
 
+	"github.com/uber/cadence/common/auth"
 	"github.com/uber/cadence/common/log"
+	"github.com/uber/cadence/common/messaging"
 	"github.com/uber/cadence/common/metrics"
 )
 
 type (
 	// This is a default implementation of Client interface which makes use of uber-go/kafka-client as consumer
 	kafkaClient struct {
```

Comment:
nit: as the package is named kafka, maybe you can just use client here

Reply:
ok